src/pysolid/solid.for

```diff
@@ -8,35 +8,30 @@
 ***     dependent subroutines can be download from IERS conventions website as:
 ***     wget -r -l1 --no-parent -R "index.html*" -nH --cut-dirs=3 https://iers-conventions.obspm.fr/content/chapter7/software/dehanttideinel
 *** Z. Yunjun and S. Sangha, Sep 2020: modify solid() to solid_point/grid() as subroutines.
-
-      subroutine solid_grid(txt_file,iyr,imo,idy,ihh,imm,iss,
-     * glad0,steplat,nlat,
-     * glod0,steplon,nlon)
-
+      subroutine solid_grid(iyr,imo,idy,ihh,imm,iss,
+     * glad0,steplat,nlat,glod0,steplon,nlon,output)
+ 
 *** calculate solid earth tides (SET) for one spatial grid given the date/time
-*** Arguments: txt_file                - string, output file name
-***            iyr/imo/idy/ihh/imm/iss - int, date/time for YYYY/MM/DD/HH/MM/SS
+*** Arguments: iyr/imo/idy/ihh/imm/iss - int, date/time for YYYY/MM/DD/HH/MM/SS
 ***            glad0/glad1/steplat     - float, north(Y_FIRST)/south/step(negative) in deg
 ***            glod0/glod1/steplon     - float, west(X_FIRST) /east /step(positive) in deg
-*** Returns:   latitude,  longitude,  SET_east,  SET_north,  SET_up
+*** Returns:   output: 3D array with shape (nlat, nlon, 5)
+***            each depth slice: [SET_east,  SET_north,  SET_up]
```

Comment:
How about returning 3 arrays of `tide_e, tide_n, tide_u` instead, for both `solid_grid` and `solid_point`? That would be simpler and more clear.